net/netns/netns.go

```diff
@@ -19,12 +19,23 @@ import (
 	"net"
 
 	"inet.af/netaddr"
+	"tailscale.com/syncs"
 )
 
+var disabled syncs.AtomicBool
+
+// Disable disables netns for the process.
+func Disable() {
```

Comment:
Recommend we log that netns is being disabled.

Reply:
We already log that userspace-networking is being used, though? They're 1:1 so it doesn't strictly add any information.

I don't really mind but I'm trying to be a little bit more cautious of adding lots of logs.

Reply:
+1

Reply:
yeah, right before this is:

```
                logf("wgengine.NewUserspaceEngine(tun %q) ...", name)
```

Where the `name` is `"userspace-networking"`

Reply:
Or did you want to backport this to 1.14.x?

Reply:
I wasn't going to recommend that, the people running into it so far would likely be fine trying an unstable release.